pkg/server/ingest/entryHandlers.go

```diff
@@ -33,11 +37,49 @@ import (
 	"github.com/siglens/siglens/pkg/otlp"
 	"github.com/siglens/siglens/pkg/sampledataset"
 	serverutils "github.com/siglens/siglens/pkg/server/utils"
+	log "github.com/sirupsen/logrus"
 	"github.com/valyala/fasthttp"
 )
 
+var diskUsageExceeded int32
+
+func MonitorDiskUsage() {
+	for {
+		usage, err := getDiskUsagePercent()
+		if err != nil {
+			log.Error("MonitorDiskUsage: Error getting disk usage:", err)
+			time.Sleep(30 * time.Second)
+			continue
+		}
+		if usage >= config.GetDataDiskThresholdPercent() {
+			log.Infof("MonitorDiskUsage: Disk usage exceeded the threshold of %v", config.GetDataDiskThresholdPercent())
```

Comment:
log this as error.
Also it would be helpful not only display the threshold but also the actual usage. Threshold anyways can be found from the config but the running usage value can be know only in this function